lib/install/validate/validator.go

```diff
@@ -56,13 +58,25 @@ type Validator struct {
 	Session *session.Session
 	Context context.Context
 
-	isVC   bool
-	issues []error
+	isVC           bool
+	firewallIssues []error
```

Comment:
I would rather not to add a specific issues type here. Following comments will show how to do.